source/MaterialXRender/Handlers/stbImageLoader.cpp

```diff
@@ -0,0 +1,98 @@
+#include <MaterialXRender/Window/HardwarePlatform.h>
+
+#if defined(OSWin_)
+#pragma warning( push )
+#pragma warning( disable: 4100)
```

Comment:
Need to clobber 1 warning on Windows. 4100 = unreferenced parameter so is harmless.
Does not show up on Linux or Mac.